src/main/resources/application.conf

```diff
@@ -27,4 +27,7 @@ akka {
       port = 2552
     }
   }
+  cluster {
```

Comment:
could you also remove above netty.tcp config

Reply:
#60